lib/completion.dart

```diff
@@ -63,7 +63,7 @@ class DartCompleter extends CodeCompleter {
       List<Completion> completions =  analysisCompletions.map((completion) {
         // TODO: Move to using a LabelProvider; decouple the data and rendering.
         String displayString = completion.isMethod
-            ? '${completion.text}()' : completion.text;
+            ? completion.text + completion.parameters : completion.text;
```

Comment:
This is pretty frowned on by some in Dart land, could you make it "${completion.text} ${completion.parameters}"